### PR TITLE
Feature/rewrite aestetik lightning

### PR DESCRIPTION
**Pull request by lakii3741**
Main class components
**AESTETIK:** Provides a scikit-learn-like interface for the AESTETIK model
**AESTETIKDataModule:** Prepares input data for the autoencoder.
1. Computes separate clusters for transcriptomics and morphology features
2. Construct input grids

**AESTETIKModel:** Implements the core AESTETIK model using PyTorch Lightning.

### CODE REVIEW

**Comment by KalinNonchev** [comment]
2. Could you also include similar functionality to:
> model.vizualize(plot_loss=True) 

for debugging purposes. You would need to store the loss during training and then plot it using the helper functions in `utils_vizualization`

**Comment by KalinNonchev** [comment]
3. Once you have implemented it, please also include it in the Getting Started notebooks with a short explanation.

**Comment by KalinNonchev** [comment]
4. Please also extend the README by including a changelog section similar to https://github.com/KalinNonchev/gnomAD_DB and very briefly outlining your changes in the log

**Comment by lakii3741** [comment]
This pull request replaces the legacy AESTETIK implementation with a reimplementation using Pytorch Lightning. This reimplementation aims to:
•	Provide an API similar to scikit-learn conventions
•	Implement a more modular architecture
•	Extend functionality to support multiple samples

**Verification**
To validate the new implementation (AESTETIK_16062025), its performance was compared against the old AESTETIK_legacy version. The validation was performed on the LIBD Maynard human brain dataset across multiple parameter configurations.
For each fixed parameter configuration (e.g. morphology_weight = 0, window_size = 5, refine_cluster = 0), we ran the following procedure:
1.	Run AESTETIK_legacy and AESTETIK_16062025 on fixed parameter set with 5 different seeds 2021-2025.
2.	Compute mean ARI and standard deviation for each of the 12 sample across the 5 seeds.

**Results**
The plot below shows an example of such a comparison. 
mw0_ws5_rc0

**Comment by KalinNonchev** [comment]
Thank you @lakii3741
